ansible/roles/k8s_cluster/tasks/install_containerd.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Copy containerd containerd.conf file
+  template:
+    src: "containerd.conf.j2"
+    dest: "/etc/modules-load.d/containerd.conf"
+  become: yes
+
+- name: "modprobe overlay"
+  shell: "modprobe overlay"
+  become: yes
+
+- name: "modprobe br_netfilter"
+  shell: "modprobe br_netfilter"
+  become: yes
+
+- name: Ensure no old docker version is installed
+  yum:
+    name: "{{ docker_old_packages }}"
+    state: absent
+  become: yes
+
+- name: Install dependencies needed by containerd
+  package:
+    name: "{{ containerd_dependencies }}"
+    state: present
+  become: yes
+
+- name: Add docker repository
+  yum_repository:
+    name: docker
+    description: Docker
+    baseurl: https://download.docker.com/linux/centos/7/$basearch/stable
+    enabled: 1
+    gpgcheck: 1
+    gpgkey: https://download.docker.com/linux/centos/gpg
+  become: yes
+
+- name: Install containerd
+  package:
+    # name: docker-ce-18.06.2.ce
+    # name: docker-ce
+    name: "{{ containerd_packages }}"
+    state: present
+  become: yes
+
+- name: Ensures /etc/containerd dir exists
+  file: 
+    path: "/etc/containerd"
+    state: "directory"
+  become: yes
+
+- name: "Configure containerd"
+  shell: "containerd config default > /etc/containerd/config.toml"
+  become: yes
+
+- name: "Set root folder"
+  lineinfile:
+    path: "/etc/containerd/config.toml"
+    regexp: '^root = "/var/lib/containerd"'
+    line: 'root = "{{ containerd_root_dir }}"'
+  when: "containerd_root_dir is defined"
+  become: yes
+
+# - name: "Enable systemd cgroug"
+#   lineinfile:
+#     path: "/etc/containerd/config.toml"
+#     regexp: '^    systemd_cgroup = false'
+#     line: "    systemd_cgroup = true"
+#   become: yes
```

Comment:
Why is it commented ?

Reply:
Because I didn't need that on my last installation, CentOS 7. I didn't remove it because I'm not sure it is required at later versions.

Reply:
Can you then document why it is commented within the code please ?

Reply:
I don't have enough information so I removed the commented block.